File: src/common/Helpers/GitHubTokenExpiredException.cs

/// <summary>
/// Exception thrown when GitHub token has expired and needs re-authentication
/// </summary>
public class GitHubTokenExpiredException : InvalidOperationException
{
    public GitHubTokenExpiredException(string message) : base(message) { }
    public GitHubTokenExpiredException(string message, Exception innerException) : base(message, innerException) { }
}

File: src/commonjs/Helpers/GitHubTokenExpiredException.ts

/**
 * Exception thrown when GitHub token has expired and needs re-authentication
 */
export class GitHubTokenExpiredException extends Error {
    public constructor(message: string);
    public constructor(message: string, innerException: Error);
    public constructor(message: string, innerException?: Error) {
        super(message);
        this.name = 'GitHubTokenExpiredException';
        if (innerException) {
            this.cause = innerException;
        }
    }
}